cli/Cargo.toml

```diff
@@ -22,13 +22,13 @@ itertools = "0.8"
 log = "0.4"
 heck = "0.3"
 futures = "0.1.28"
-jsonrpc-core-client = { version = "13.0", features = ["ws"] }
+jsonrpc-core-client = { version = "14.0", features = ["ws"] }
 zip = { version = "0.5", default-features = false }
 pwasm-utils = "0.11.0"
 parity-wasm = "0.40.2"
 cargo_metadata = "0.8.2"
 substrate-primitives = { git = "https://github.com/paritytech/substrate/", package = "substrate-primitives" }
-subxt = { git = "https://github.com/paritytech/substrate-subxt/", branch = "v0.1", package = "substrate-subxt" }
+subxt = { git = "https://github.com/paritytech/substrate-subxt/", branch = "master", package = "substrate-subxt" }
```

Comment:
Not sure if we really want to be using `master` because that sounds rather unstable to me.

Reply:
Once the `substrate-subxt` v0.1 branch is updated, I will revert it.

Reply:
@ascjones comment needed

Reply:
Branch `v0.1` updated